manifests/base/deployment.yaml

```diff
@@ -118,6 +118,8 @@ spec:
             name: oauth-config
           - mountPath: /etc/oauth/client
             name: oauth-client
+          - mountPath: /etc/pki/tls/certs
+            name: trusted-ca-bundle
```

Comment:
noticed the same is done in the dashboard container, is this not overwriting the default `/etc/pki/tls/certs` path?

did this work in ROSA?

Reply:
well yes, but we are explicitly adding the CAs that we want in Node, we ignore everything else, do you have any other suggestion @HumairAK ?

Reply:
Ok, I've updated the PR based on the discussion thread, thanks a lot